tests/sandbox/linalg/test_linalg.py

```diff
@@ -152,3 +152,27 @@ def test_matrix_inverse_solve():
     node = matrix_inverse(A).dot(b).owner
     [out] = inv_as_solve.transform(None, node)
     assert isinstance(out.owner.op, Solve)
+
+
+def test_cholesky_dot_lower():
+    cholesky_lower = Cholesky(lower=True)
+
+    L = matrix("L")
+    L.tag.lower_triangular = True
+
+    C = cholesky_lower(L.dot(L.T))
+    f = pytensor.function([L], C)
+    if config.mode != "FAST_COMPILE":
+        assert f.maker.fgraph.outputs[0].name == "L"
```

Comment:
We don't make any promises of keeping variables names around, so this test would be more sensible like this
```suggestion
        assert f.maker.fgraph.outputs[0] == f.maker.fgraph.inputs[0]
```

Same for the other test

Reply:
Sounds good. I noticed we were going to get deep copy so followed `test_transinv_to_invtrans` above. Maybe `test_transinv_to_invtrans` should be changed too? Anyway fixed this for now.